_posts/2021-02-11-notes-from-community-call-4-february-2021.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Notes from the Standard for Public Code community call - 4 February 2021"
+date: 2021-02-11
+author: Jan Ainali
+type: blogpost
+excerpt: How will certifications look like?
+categories:
+  - Community call
+---
+
+# Notes from the Standard for Public Code community call - 4 February 2021
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/team/eric-herman.html)
+* [Elena Findley-de Regt](https://publiccode.net/team/elena-findley-de-regt.html)
+* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)
+
+## Updates from the Foundation
+
+* Signalen is incubation for the Standard for Pubic Code
+* Signalen's and OpenZaak's assessments are both updated to the v.0.2.1 of the template
```

Comment:
please hyperlink [v.0.2.1 of the template]

Reply:
Do we want to link to the empty template on our site, or each of the assessments?

Reply:
Empty template!